asdf/tags/core/ndarray.py

```diff
@@ -408,32 +411,56 @@ def reserve_blocks(cls, data, ctx):
 
     @classmethod
     def to_tree(cls, data, ctx):
+        # The ndarray-1.0.0 schema does not permit 0 valued strides.
+        # Perhaps we'll want to allow this someday, to efficiently
+        # represent an array of all the same value.
```

Comment:
Isn't the `ndarray-1.0.0` invalid?  What about `ndarray-1.1.0`? I.e. `test_example2`.

Reply:
We haven't created `ndarray-1.1.0` yet, have we?  I don't see it in asdf-standard...

Reply:
Doh!  Nevermind.

I guess this would be a candidate feature of 1.1.0.  =)